tests/stream-properties.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<script src="resources/text-encode-transform.js"></script>
+<script src="resources/transform-stream.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+'use strict';
+
+test(() => {
+  const te = new TextEncoder();
+  assert_idl_attribute(te, 'readable', 'readable property must be present');
```

Comment:
sorry. it's first time for me to see this assert tool. could you point me to some doc to understand how this works?

http://web-platform-tests.org/writing-tests/testharness-api.html and https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/resources/testharness.js?type=cs&q=assert_idl_attribute&sq=package:chromium&l=1227 don't tell me much.

Reply:
Based on the source, it is just an alias for assert_inherits(). I was hoping it would check that the property is enumerable, since the new implementation fixes that, but it doesn't. I could add a check for enumerability in a follow-up change.

Reply:
Got it. Sounds good.